snapshot/lib/snapshot/test_command_generator_xcode_8.rb

```diff
@@ -0,0 +1,63 @@
+require 'snapshot/test_command_generator_base'
+
+module Snapshot
+  # Responsible for building the fully working xcodebuild command
+  class TestCommandGeneratorXcode8 < TestCommandGeneratorBase
```

Comment:
Maybe some comments here and in the other classes explaining the differences between how it works with Xcode 8 and Xcode 9